drivers/si70xx/si70xx.c

```diff
@@ -39,11 +39,11 @@ static uint16_t _do_measure(const si70xx_t *dev, uint8_t command)
 
     i2c_acquire(SI70XX_I2C);
 
-    if (i2c_write_byte(SI70XX_I2C, SI70XX_ADDR, command) != 1) {
```

Comment:
I2C_ACK doesn't exist anymore as we switch to errno used. This must be replaced by 0.